extensions/python/tests/ExecutePythonProcessorTests.cpp

```diff
@@ -221,95 +186,29 @@ class SimplePythonFlowFileTransferTest : public ExecutePythonProcessorTestBase {
 //
 TEST_CASE_METHOD(SimplePythonFlowFileTransferTest, "Simple file passthrough", "[executePythonProcessorSimple]") {
   // Expectations
-  const auto OUTPUT_FILE_MATCHES_INPUT          = SimplePythonFlowFileTransferTest::Expectation::OUTPUT_FILE_MATCHES_INPUT;
-  const auto RUNTIME_RELATIONSHIP_EXCEPTION    = SimplePythonFlowFileTransferTest::Expectation::RUNTIME_RELATIONSHIP_EXCEPTION;
+  const auto OUTPUT_FILE_MATCHES_INPUT = SimplePythonFlowFileTransferTest::Expectation::OUTPUT_FILE_MATCHES_INPUT;
+  const auto RUNTIME_RELATIONSHIP_EXCEPTION = SimplePythonFlowFileTransferTest::Expectation::RUNTIME_RELATIONSHIP_EXCEPTION;
+  const auto INITIALIZATION_EXCEPTION = SimplePythonFlowFileTransferTest::Expectation::INITIALIZATION_EXCEPTION;
   // ExecutePython outbound relationships
-  const core::Relationship SUCCESS {"success", "description"};
+  const core::Relationship SUCCESS{"success", "description"};
   const core::Relationship FAILURE{"failure", "description"};
 
-  // For the tests "" is treated as none-provided since no optional implementation was ported to the project yet
-
   // 0. Neither valid script file nor script body provided
-  //                                      TEST EXPECTATION  OUT_REL        USE_AS_SCRIPT_FILE  USE_AS_SCRIPT_BODY
-  testSimpleFilePassthrough(RUNTIME_RELATIONSHIP_EXCEPTION, SUCCESS,                       "", "");  // NOLINT
-  testSimpleFilePassthrough(RUNTIME_RELATIONSHIP_EXCEPTION, FAILURE,                       "", "");  // NOLINT
-  testSimpleFilePassthrough(RUNTIME_RELATIONSHIP_EXCEPTION, SUCCESS, "non_existent_script.py", "");  // NOLINT
-  testSimpleFilePassthrough(RUNTIME_RELATIONSHIP_EXCEPTION, FAILURE, "non_existent_script.py", "");  // NOLINT
+  //                                      TEST EXPECTATION  OUT_REL        USE_AS_SCRIPT_FILE
+  testSimpleFilePassthrough(INITIALIZATION_EXCEPTION, SUCCESS, "non_existent_script.py");  // NOLINT
+  testSimpleFilePassthrough(INITIALIZATION_EXCEPTION, FAILURE, "non_existent_script.py");  // NOLINT
```

Comment:
very minor, but the header is out of alignment:
```suggestion
  //                                TEST EXPECTATION  OUT_REL        USE_AS_SCRIPT_FILE
  testSimpleFilePassthrough(INITIALIZATION_EXCEPTION, SUCCESS, "non_existent_script.py");  // NOLINT
  testSimpleFilePassthrough(INITIALIZATION_EXCEPTION, FAILURE, "non_existent_script.py");  // NOLINT
```

Reply:
Updated in https://github.com/apache/nifi-minifi-cpp/pull/2039/commits/12ef8790752607b63dc531f81c16bdf393cfa88d